tests/components/nibe_heatpump/test_climate.py

```diff
@@ -145,6 +147,7 @@ async def test_set_temperature(
     model: Model,
     climate_id: str,
     entity_id: str,
+    support_cooling: bool,
```

Comment:
Please silly this test in two. One for the supported case one for the unsupported

Reply:
I have split up in two test. A simple unsupported cooling that just sets heating and then tries to set cooling and the original set_temperature test.